docs/intro.mdx

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
why is this file an `mdx` file, vs others that are `md`? is there something special about this file where it needs mdx?